pulp_2_tests/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -612,3 +613,72 @@ def test_zchunk_sync(self):
             RPM_ZCHUNK_FEED_COUNT,
             copied_unit_ids
         )
+
+
+class YumMetadataSymlinkTesCase(unittest.TestCase):
+    """Yum metadata symlink test case.
+
+    This test targets the following issues:
+
+    * `Pulp #4560 https://pulp.plan.io/issues/4560`_
+    * `Pulp #4632 https://pulp.plan.io/issues/4632`_
+
+    It does the following:
+
+    1. Sync, and publish a repository with yum metadata.
+    2. Verify that the published yum metadata is not a symlink.
+    """
```

Comment:
```suggestion
    3. Verify productid was saved as a file
    """
```

Reply:
@kersommoura technically, don't you have another verification for `yum_repo_metadata file present in the repo.`? Might as well add that, no?

Reply:
Talked locally. That is covered by 1.